### ISSUE TITLE
Invisible Domain on Tracking List

### ISSUE THREAD

**Issue by ablanathtanalba** [EFForg/privacybadger#2324]
Looks like this is related to: https://github.com/EFForg/privacybadger/issues/550

The first domain in my tracking list is an empty string. If it's related to that issue from a few years back, it may have to do with the number of tracking domains (mine here is about 2000). Unlike what one user describes in that original issue, I am able to move the slider in different directions on any of the domains in this list (although they often jump out of frame and move to another part of the list when done so, that can be filed in a separate issue.)

<img width="874" alt="screen shot 2019-03-07 at 10 48 19 pm" src="https://user-images.githubusercontent.com/25778052/54006660-42682a00-412c-11e9-86c3-9336b866128b.png">

**Comment by ghostwords** [comment]
Could you run the following script in your Badger's background page console and share what it prints out?
```javascript
(function () {
  console.log("**** ACTION_MAP for ''");
  console.log(JSON.stringify(badger.storage.getBadgerStorageObject('action_map')._store[""], null, 2))
  console.log("**** SNITCH_MAP for ''");
  console.log(JSON.stringify(badger.storage.getBadgerStorageObject('snitch_map')._store[""], null, 2))
}());
```

**Comment by ghostwords** [comment]
Which browser do you use?

**Comment by ghostwords** [comment]
If Firefox, it's probably https://github.com/EFForg/privacybadger/pull/2167#issuecomment-425260296.

**Comment by ablanathtanalba** [comment]
> Which browser do you use?

Chrome 72.0.3626.121

**Comment by ablanathtanalba** [comment]
> Could you run the following script in your Badger's background page console and share what it prints out?
> 
> ```js
> (function () {
>   console.log("**** ACTION_MAP for ''");
>   console.log(JSON.stringify(badger.storage.getBadgerStorageObject('action_map')._store[""], null, 2))
>   console.log("**** SNITCH_MAP for ''");
>   console.log(JSON.stringify(badger.storage.getBadgerStorageObject('snitch_map')._store[""], null, 2))
> }());
> ```

<img width="257" alt="Screen Shot 2019-03-10 at 5 37 30 PM" src="https://user-images.githubusercontent.com/25778052/54091841-504fc200-435b-11e9-98ab-bf53c7b62dd3.png">
Cool trick!

**Comment by ablanathtanalba** [comment]
I just tested this out with a new badger on Firefox 65.0.1 (64bit) and trained the badger to pick up on about 1100 tracking domains, but was unable to recreate the same kind of phantom URL.

**Comment by ghostwords** [comment]
Ah, if this is Chrome and the site domain is `twitter.com`, then the underlying issue was fixed in #2205. I guess we didn't do a migration to remove the leftover blank domain. You could remove it manually now if you wish. Sorry for the missed nit!

**Comment by ghostwords** [comment]
Duplicate of #1447.
